core/src/test/java/org/mskcc/cbio/portal/util/TestValueParser.java

```diff
@@ -64,7 +64,7 @@ public void testValueParser() {
       assertFalse(parser.isCnaGained());
       assertTrue(parser.isMutated());
       assertTrue(parser.isGeneAltered());
-      // default CNA alteration is AMP and HOMDEL
```

Comment:
Tests for a file I don;t think we should change, so let's not change these either.

Reply:
done